### PR TITLE
Tell accountmanager url changed

### PR DESCRIPTION
**Pull request by ksuprynowicz**
Branch made by Frost(Mirrorwolf), that will allow live switching of current metaverse server. Currently in this branch switching servers requires restart of the interface, so more work is needed.

### CODE REVIEW

**Comment by ksuprynowicz** [comment]
When changing metaverse server, account manager setAuthURL method apparently fails to be called:
`[01/22 16:24:42] [WARNING] [default] qrc:/qml/LoginDialog/LinkAccountBody.qml:410: ReferenceError: setAuthURL is not defined`
I think this is the reason that live switching fails.

**Comment by ksuprynowicz** [comment]
I tested it on Linux an it seems to work well now. I need a code review :)

**Comment by ksuprynowicz** [comment]
It's tested on Windows now too.

**Review by daleglass — CHANGES_REQUESTED** [review]

File: interface/resources/qml/LoginDialog/LinkAccountBody.qml [hunk 1]
```diff
@@ -74,6 +74,19 @@ Item {
...
+        // make sure the metaverse server is set so we don't send your password to the wrong place!
+        if (!isLoggingInToDomain) {
+            Settings.setValue("private/selectedMetaverseURL", metaverseServerField.text);
+        }
+        
+        if (keepMeLoggedInCheckbox.checked) {
+            if (!isLoggingInToDomain) {
+                Settings.setValue("keepMeLoggedIn/savedUsername", emailField.text);
+            } else {
+                // ####### TODO
```
[the remark this answers is not in the document]
Comment (ksuprynowicz) [reply interface/resources/qml/LoginDialog/LinkAccountBody.qml:86]: For security I added clearing pasword on every change of the metaverse server field.

**Review by daleglass — APPROVED** [review]

**Pull request merged**